src/components/shared/tech-input.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from '@styled-components';
+import { theme } from '@styles/theme';
+
+interface Props {
+  value: string;
+}
+
+const Tech = styled.span`
+  background: ${theme.colors.highlight};
```

Comment:
To use provided theme:
```suggestion
  background: ${({ theme }) => theme.colors.highlight};
```

Reply:
this file is not used anymore. file has been deleted.